plugins/generator-1.21.1/neoforge-1.21.1/mappings/attributes.yaml

```diff
@@ -0,0 +1,33 @@
+_default: Attributes.ARMOR.getDelegate()

```

Comment:
Are you sure getDelegate is needed? Checking vanilla code, it always operates on direct instances and not on a delegate (delegates are actually (Neo)Forge addition)

![image](https://github.com/user-attachments/assets/0de95901-2a6c-409c-8987-37f0c7deef61)

Reply:
Ok it seems that in 1.21 it is no longer needed